### PR TITLE
Implement "Playground"

### PR DESCRIPTION
**Pull request by dhonx**
### Implement Playground (`v.playground` command).

This feature will get the contents of the currently opened file and open it in [DevBit Playground](https://devbits.app/play). 
V Playground is currently unstable and there are no features available to allow us to pass the encoded code through the URL parameter. Thanks to [DevBit](https://devbits.app) that accepting my suggestion to add this feature.  🙂

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -109,7 +109,7 @@
 			},
 			{
 				"command": "v.playground",
-				"title": "V: Upload and share current code to V playground"
+				"title": "V: Open current file on DevBits V playground"
```
Comment (dhonx) [inline package.json]: I'm still unsure of the name of this label. what do you think guys? 🤔

**Comment by Delta456** [comment]
I think you should give an option on choosing the Playground instead of one option.

**Comment by dhonx** [comment]
Good idea! but after V playground add a feature that allows us to pass the encoded code through the URL parameter. There is no method to send data to the V playground.

File: package.json [hunk 2]
```diff
@@ -109,7 +109,8 @@
 			},
 			{
 				"command": "v.playground",
-				"title": "V: Upload and share current code to V playground"
+				"title": "V: Open current file on DevBits V playground",
+				"enablement": "editorLangId == v"
```
Comment (alexesprit) [inline package.json]: I switched to `when` property in another PR, so it can be removed in favor of `when`.

Comment (dhonx) [reply package.json]: Yeah... I notice that. will push later.

**Comment by dhonx** [comment]
Done!!

File: package.json [hunk 3]
```diff
@@ -150,7 +150,7 @@
 	},
 	"devDependencies": {
 		"@types/node": "^13.1.1",
-		"@types/vscode": "~1.20.0",
+		"@types/vscode": "^1.41.0",
```
Comment (dhonx) [inline package.json]: We need the newer version of `@types/vscode`. Also, this PR using the newer API of `vscode`.

Comment (0x9ef) [reply package.json]: If this does not kill compatibility with older versions of VSCode, then okay.

Comment (dhonx) [reply package.json]: No... it's a new API.

**Pull request closed without merging**